site/en/docs/lighthouse/accessibility/custom-control-roles/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Custom controls have ARIA roles
+description: |
+  Learn how to improve your web page's accessibility by making sure custom
+  controls have ARIA roles that assistive technologies can interpret.
+date: 2019-05-02
+updated: 2019-09-19
+web_lighthouse:
+  - custom-control-roles
+---
+
+Check that all custom controls have an appropriate `role` and
+any required ARIA attributes that confer their properties and state.
+For example, a custom checkbox needs a `role="checkbox"` and
+`aria-checked="true|false"` to properly convey its state.
+See the [Introduction to ARIA](https://web.dev/semantics-aria/)
+for a general overview of how ARIA can provide missing semantics for custom controls.
+
+## How to manually test
+
+To check that all custom interactive controls have appropriate ARIA roles,
+test the page using either the
+[Chrome DevTools accessibility pane](/docs/devtools/accessibility/reference/#pane)
+or a screen reader.
+[JAWS](https://www.freedomscientific.com/products/software/jaws/) and
+[NVDA](https://www.nvaccess.org/)
+are two of the more popular screen readers for Windows.
+[VoiceOver](https://www.apple.com/accessibility/mac/vision/)
+is the screen reader built into MacOS.
+
+Using CSS, it's possible
+to style the `<div>` and `<button>` elements so they convey the same visual affordances,
+but the two experiences are very different when using a screen reader.
+A `<div>` is just a generic grouping element,
+so a screen reader only announces the text content of the `<div>`.
+The `<button>` is announced as a "button,"
+a much stronger signal to the user that it's something they can interact with.
+See also [Semantics and screen readers](https://web.dev/semantics-and-screen-readers/).
+
+## How to fix
+
+The simplest
+and often best solution to this problem
+is to avoid custom interactive controls altogether.
+For example, replace a `<div>` that's acting like a button
+with an actual `<button>`.
+
+If you must keep the `<div>`,
+then add `role="button"` and `aria-pressed="false"` to the `<div>`:
+
+{% Glitch {
+  id: 'gorgeous-raven',
+  path: 'example.html:13:39',
+  height: 346
+} %}
```

Comment:
unfortunately several of these rely on a hosted form of chromevox-lite ([in this example](https://gorgeous-raven.glitch.me/) at `https://robdodson.github.io/chromevox-lite/dist/index.min.mjs`) which doesn't exist in that repo anymore and doesn't seem to be hosted elsewhere now. If we can't find a replacement, I guess we'll just delete?

Reply:
I suppose if that's what we need to do until we can find an interim solution. Is there a way to comment out nunjucks syntax? Perhaps it could be fixed later.

Reply:
> I suppose if that's what we need to do until we can find an interim solution. Is there a way to comment out nunjucks syntax? Perhaps it could be fixed later.

Commented it out and left an short explanation.

I was wrong and the other glitches are actually fine, it's just this one, so it's not a huge deal if we can't get a replacement.